streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -137,10 +153,16 @@ public synchronized final TopologyBuilder setApplicationId(final String applicat
      * {@link #addProcessor(String, ProcessorSupplier, String...) adding processor children}.
      * @param topics the name of one or more Kafka topics that this source is to consume
      * @return this builder instance so methods can be chained together; never null
+     * @deprecated use {@link Topology} instead
      */
+    @Deprecated
```

Comment:
Do we need to add `@SuppressWarnings("deprecation")` here, too?

And elsewhere in this class